### DISCUSSION TITLE
Custom fields for Manual Payment Plugin

### DISCUSSION THREAD

**Discussion by naveed-strativ** [pretix/pretix#4602] [Q&A] [answered]
Hello,
I am trying the Manual Payment Plugin, but although the plugin description says "_A fully customizable payment method for manual processing_", I can neither add any custom fields nor make any fields mandatory from the "_manual payment settings_". 

Is this how its supposed to work? Or am I missing something?

_Screenshot of the plugin_
<img width="1135" alt="image" src="https://github.com/user-attachments/assets/550802b8-a4ce-4ff0-bba5-f35d79a29df7">

**Comment by raphaelm** [comment, marked as answer]
Okay, maybe the "fully" in "fully customizable" was a little optimistic there ;) The idea is that you can do whatever you want *outside* of the system and all texts are fully customizable.

For example, a common use case looks like this:

- Payment method name: "Payment by cash"
- Payment method description: "To pay your order, come by our office Monday–Friday 9-5 and pay in cash."

There is no way to add custom form fields currently.

**Reply by naveed-strativ** [reply]
Ahh I see, makes sense now. 
Do you guys have any plans to make the _"add custom fields/make a field mandatory"_ to be a feature in the future?

**Reply by raphaelm** [reply]
Not yet, it would probably require quite some changes to how payment methods are configured.
